pkg/sidecar/config.go

```diff
@@ -120,21 +124,22 @@ func ValidateConfig(c *Config) error {
 		c.RenewSignal = c.RenewSignalDeprecated
 	}
 
+	for _, jwtConfig := range c.JwtSvids {
+		if countEmpty(jwtConfig.JWTSvidFilename, jwtConfig.JWTAudience) > 0 {
+			return errors.New("both 'jwt_file_name' and 'jwt_audience' are required in 'jwt_svids'")
+		}
+	}
+
 	x509EmptyCount := countEmpty(c.SvidFileName, c.SvidBundleFileName, c.SvidKeyFileName)
-	jwtSVIDEmptyCount := countEmpty(c.JWTSvidFilename, c.JWTAudience)
 	jwtBundleEmptyCount := countEmpty(c.SvidBundleFileName)
-	if x509EmptyCount == 3 && jwtSVIDEmptyCount == 2 && jwtBundleEmptyCount == 1 {
-		return errors.New("at least one of the sets ('svid_file_name', 'svid_key_file_name', 'svid_bundle_file_name'), ('jwt_file_name', 'jwt_audience'), or ('jwt_bundle_file_name') must be fully specified")
+	if x509EmptyCount == 3 && c.JwtSvids == nil && jwtBundleEmptyCount == 1 {
+		return errors.New("at least one of the sets ('svid_file_name', 'svid_key_file_name', 'svid_bundle_file_name'), ('jwt_file_name', 'jwt_audience'), 'jwt_svids', or ('jwt_bundle_file_name') must be fully specified")
```

Comment:
looks like ('jwt_file_name', 'jwt_audience'),  are no longer configurable?

```suggestion
		return errors.New("at least one of the sets ('svid_file_name', 'svid_key_file_name', 'svid_bundle_file_name'), 'jwt_svids', or ('jwt_bundle_file_name') must be fully specified")
```

Reply:
Yes, I missed this cleanup after I deleted those options rather than deprecating them. Fixed.